c2rust-ast-exporter/src/AstExporter.cpp

```diff
@@ -2562,7 +2562,7 @@ static std::vector<const char *> augment_argv(int argc, const char *argv[]) {
     SmallString<128> P("-extra-arg=-resource-dir=" CLANG_BIN_PATH);
     llvm::sys::path::append(P, "..", Twine("lib") + CLANG_LIBDIR_SUFFIX,
                             "clang", CLANG_VERSION_STRING);
-    std::string resource_dir = P.str();
+    std::string resource_dir = P.str().str();
```

Comment:
Same as above: wrap in a check on `CLANG_VERSION_MAJOR`.

Reply:
I compiled this in a ubuntu bionic container with clang-6 and libclang-6:

```
root@71976200ca48:/src/c2rust# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
root@71976200ca48:/src/c2rust# clang --version
clang version 6.0.0-1ubuntu2 (tags/RELEASE_600/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
root@71976200ca48:/src/c2rust# cargo --version
cargo 1.41.0-nightly (626f0f40e 2019-12-03)
root@71976200ca48:/src/c2rust# rustc --version
rustc 1.41.0-nightly (6d77e45f0 2019-12-04)
root@71976200ca48:/src/c2rust# git describe  
0.1.0-3140-g8e7131e6
```

The test suite passed:
```
root@71976200ca48:/src/c2rust# python3 ./scripts/test_translator.py --debug ./tests
[...]
Test summary:                                                            
  unexpected failures: 0                                                 
  unexpected successes: 0                                                
  expected failures: 3                                                   
  successes: 108
```

I don't think an extra ifdef is necessary, but it is of course your call!

Reply:
> I don't think an extra ifdef is necessary, but it is of course your call!

It's fine, no ifdef needed.